pkg/api/routes.go

```diff
@@ -38,6 +38,7 @@ const (
 	RN_SetWorkflowVariable         = "setWorkflowVariable"
 	RN_ListNamespaceVariables      = "listNamespaceVariables"
 	RN_GetNamespaceVariable        = "getNamespaceVariable"
+	RN_GetNamespaceLogs            = "GetNamespaceLogs"
```

Comment:
This should be changed to match the rest of the RN_ constants: `getNamespaceLogs`.